tracks/bash/exercises/pangram/mentoring.md

```diff
@@ -0,0 +1,42 @@
+## Common Suggestions
+
+* use brace expansion
+
+```md
+On LN, consider a more concise way to provide a sequence of letters.
+[Brace Expansion](https://www.gnu.org/software/bash/manual/html_node/Brace-Expansion.html)
+in the Bash Reference Manual might be useful.
+```
+
+## General Guidelines
+
+* there is no `main` function
+
+```md
+Generally, you should encapsulate the main body of your script in a `main`
+function, like you may remember from the Hello World exercise.  It
+encapsulates a chunk of logic in one function which is good practice for when
```

Comment:
If we use single sentence per line, it will result in cleaner diffs when this is maintained. See the [building markdown] information.

[building markdown]: https://exercism.org/docs/building/markdown